air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -66,9 +67,14 @@ proc.compute_evaluate_integrity_constraints
     padw mem_loadw.4294900074 movdn.3 movdn.3 drop drop padw mem_loadw.4294900074 drop drop padw mem_loadw.4294900002 movdn.3 movdn.3 drop drop padw mem_loadw.4294900150 movdn.3 movdn.3 drop drop ext2add ext2mul ext2sub
     # Multiply by the composition coefficient
     padw mem_loadw.4294900202 movdn.3 movdn.3 drop drop ext2mul
-end # END PROC compute_evaluate_integrity_constraints
+end # END PROC compute_integrity_constraints
 
-proc.compute_evaluate_boundary_constraints
+# Procedure to compute the boundary divisor for the first row of the main trace
+# 
+# Input: [...]
+# Output: [(r_1, r_0)*, ...]
+# Where: (r_1, r_0) is one quadratic element for each constraint
+proc.compute_boundary_constraints_main_first
```

Comment:
I think the comment is wrong here (and for other similar procedures): we are computing boundary constraint numerators, not divisors here.

Reply:
ah, thanks! fixed